### DISCUSSION TITLE
Is there any expectations when the new library version will be released?

### DISCUSSION THREAD

**Discussion by alex-titarenko** [excalidraw/excalidraw#8272] [General]
It has been five months since the last release, and there have been new good features and fixes since then. Do you have any info on when we can expect the release?

**Comment by maielo** [comment]
Hey, 

we are currently working on new version of libraries :) we will probably call it marketplace. Thus not much is happening in repo. 

Reasoning for it:
- user can under their own account manage/edit their created libraries
- UI & excalidraw editor dedicated to create new library, which would allow it even to less tech savy excalidraw users
- API - so we can implement search directly from excalidraw editor
- nicer & searchable UI to find library you need

here is small sneak peak:
![image](https://github.com/user-attachments/assets/f45481b6-dbb7-48d9-90e7-4e0d4952c572)

**Reply by alex-titarenko** [reply]
Thank you for the answer. I'm sorry for misleading you; my question was about new NPM package.

**Reply by Mrazator** [reply]
Hi, we don't have an ETA yet, please check #8655 for more details.
